roles/nginxplus/files/fail2ban/nginx-f_inclusive.conf

```diff
@@ -0,0 +1,4 @@
+nginx-f_inclusive.conf
+[Definition]
+failregex = ^<HOST> .+?([&?]f(_inclusive)?%%5B.*?){8,}
```

Comment:
@kayiwa, @sandbergja and I are reviewing this together and the question we have is : have you identified patterns in our logs that come from a bot and thy are 8 facets and more?

Also in the catalog using the advanced search it's not hard for a human to create queries with 8 facets and more.

Reply:
this was based on the UNC input.